NavigationService/Navigation.cs

```diff
@@ -17,6 +17,17 @@ public class Navigation
         private CargoMonitor cargoMonitor = (CargoMonitor)EDDI.Instance.ObtainMonitor("Cargo monitor");
         private MissionMonitor missionMonitor = (MissionMonitor)EDDI.Instance.ObtainMonitor("Mission monitor");
 
+
+        private static Dictionary<string, dynamic> ServiceFilter = new Dictionary<string, dynamic>()
+        {
+            { "encoded", new {econ = new List<string>() {"High Tech", "Military"}, population = 1000000, security = new List<string>() {"Medium", "High"}, service = "Material Trader", cubeLy = 40} },
+            { "facilitator", new {econ = new List<string>(), population = 0, security = new List<string>() {"Low"}, service = "Interstellar Factors Contact", cubeLy = 25} },
+            { "manufactured", new {econ = new List<string>() {"Industrial"}, population = 1000000, security = new List<string>() {"Medium", "High"}, service = "Material Trader", cubeLy = 40} },
+            { "raw", new {econ = new List<string>() {"Extraction", "Refinery"}, population = 1000000, security = new List<string>() {"Medium", "High"}, service = "Material Trader", cubeLy = 40} },
+            { "guardian", new {econ = new List<string>() {"High Tech"}, population = 10000000, security = new List<string>()  {"High"}, service = "Technology Broker", cubeLy = 80} },
+            { "human", new {econ = new List<string>() {"Industrial"}, population = 10000000, security = new List<string>() {"High"}, service = "Technology Broker", cubeLy = 80} }
+        };
+
```

Comment:
Is population a prerequisite for stations with these services? 
(https://github.com/EDCD/FDevIDs/blob/master/How%20to%20determine%20MatTrader%20and%20Broker%20type suggests it might not be, but as you know it's hard for me to test assumptions here at the moment)

Reply:
It's more of a documented observation and it's all about reducing the number of systems in the 'GetOrFetch' call.